tests/config_test.go

```diff
@@ -36,11 +36,14 @@ import (
 
 var _ = Describe("[rfe_id:899][crit:medium][vendor:cnv-qe@redhat.com][level:component]Config", func() {
 
-	tests.FlagParse()
-	virtClient, err := kubecli.GetKubevirtClient()
-	tests.PanicOnError(err)
+	var virtClient kubecli.KubevirtClient
 
 	BeforeEach(func() {
+		var err error
+
+		virtClient, err = kubecli.GetKubevirtClient()
```

Comment:
wouldnt it be better to use tests.BeforeAll for it ?
so it wont do for each test, unless this is what you want.
if so then it repeats in more places (some already use tests.BeforeAll)

Reply:
`GetKubevirtClient` is already a singleton. See [here](https://github.com/kubevirt/kubevirt/blob/c5656f9e04169b3e2e4aa1156e4faaaf1d443fd9/staging/src/kubevirt.io/client-go/kubecli/kubecli.go#L308).

Reply:
We should remove `BeforeAll` completely from the code-base. It is not really well supported and it was a bad idea adding it.